netlify.toml

```diff
@@ -0,0 +1,7 @@
+[build]
+    base = "react/"
+    publish = "storybook-static"
```

Comment:
is this the correct publish directory? didn't we nest it in .storybook?

Reply:
Nope, storybook-static refers to the build folder which is `react/storybook-static`. The static files injected into the build are the ones we placed in `.storybook`